test/cpp/nativert/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+set(NATIVERT_TEST_ROOT ${TORCH_ROOT}/test/cpp/nativert)
+
+# Build the cpp gtest binary containing the cpp-only tests.
+set(NATIVERT_TEST_SRCS
+  ${NATIVERT_TEST_ROOT}/test_tensor_meta.cpp
```

Comment:
nit: Since I assume all the nativert tests will live here, can we please have some answer on when do they run and on which configs?